packages/waku/src/lib/builder/build.ts

```diff
@@ -432,6 +441,10 @@ const emitRscFiles = async (
           config.rscPath,
           encodeInput(input),
         );
+        // In partial mode, skip if the file already exists.
+        if (partial && existsSync(destRscFile)) {
```

Comment:
I wonder if we need to check `partial` mode here. Can we simply skip if the file exists?

Reply:
Good point. If not partial, everything is deleted anyway.